### PR TITLE
refactor(gomix): update to gomix.com

### PR DESCRIPTION
**Pull request by raisedadead**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)

#### Checklist:

- [x] Tested changes locally.

#### Description

gomix.me redirects to gomix.com, this PR updates the references.

### CODE REVIEW

File: seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
       "id": "bd7158d8c443edefaeb5bdef",
       "title": "Timestamp Microservice",
       "description": [
-        "Build a full stack JavaScript app that is functionally similar to this: <a href='https://curse-arrow.gomix.me/' target='_blank'>https://curse-arrow.gomix.me/</a> on GoMix or on your own publicly accessible domain.",
+        "Build a full stack JavaScript app that is functionally similar to this: <a href='https://curse-arrow.gomix.com/' target='_blank'>https://curse-arrow.gomix.com/</a> on GoMix or on your own publicly accessible domain.",
```
Comment (Greenheart) [inline seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json]: https://curse-arrow.gomix.com doesn't work for me. https://curse-arrow.gomix.me does.

**Review by Greenheart — CHANGES_REQUESTED** [review]
@raisedadead I don't think this is an issue.

`<my-app>.gomix.me` is the format for the publicly available projects that are hosted.

`gomix.com` is the online IDE + the official gomix website.

Visiting gomix.me without the subdomain of a valid project should redirect to the main website, which it does.

**Comment by raisedadead** [comment]
Umm.. I might have confused with some redirects earlier. Closing.

**Comment by Greenheart** [comment]
Haha no worries! Name changes always seem to cause so much confusion ^^

söndag 19 februari 2017 skrev mrugesh mohapatra <notifications@github.com>:

**Pull request closed without merging**